scripts/DeployAgTokenSideChainMultiBridge.s.sol

```diff
@@ -57,7 +57,7 @@ contract DeployAgTokenSideChainMultiBridge is Script, CommonUtils {
         address computedAddress = create2Factory.findCreate2Address(salt, initCode);
         console.log("AgTokenSideChainMultiBridge Proxy Supposed to deploy: %s", computedAddress);
 
-        // require(computedAddress == expectedAddress, "Computed address does not match expected address");
+        require(computedAddress == expectedAddress, "Computed address does not match expected address");
```

Comment:
and the expected address we can fetch it from the json for the USD

Reply:
from the sdk (Ethereum) as it should exist